actionview/CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+*   Allow templates to define which locals they accept.
+
+    For example:
+
+    ```erb
+    <%# accepts_locals(message:) -%>
+    <%= message %>
+    ```
+
+    Default values can also be provided:
+
+    ```erb
+    <%# accepts_locals(message: "Hello, world!") -%>
+    <%= message %>
+    ```
+
+    To disallow locals entirely, set `# accepts_locals()`.
```

Comment:
I guess what weirds me out a bit, is that most proposals so far use some variation of `locals` (which makes sense since it's the parameter you use to pass them), but to me this is more akin to declaring a method signature, hence why I'm more attracted by something like:

```
<%# def(message:, author: "George") %>
```

or

```
<%# sig(message:, author: "George") %>
```

But that's just my opinion, I don't feel that strongly about it.